frontend/src/components/common/SimpleTable.stories.tsx

```diff
@@ -223,31 +223,30 @@ ReflectInURLWithPrefix.args = {
 
 // filter Function
 
-type SimpleTableWithFilterProps = SimpleTableProps & { matchCriteria?: string[] };
```

Comment:
Is SimpleTable being deprecated? If so, it should be marked so.

If it's being kept for backwards compatibility it should be backwards compatible without an API change.

Reply:
SimpleTable itself is not changed and not deprecated, it can still be used for simple cases. api is kept unchanged.
The reason there are changes here is because these stories are also coupled with the filterSlice Context

Reply:
Ok, thanks.

@joaquimrocha is it your understanding that we not deprecate SimpleTable?

Reply:
I think eventually we do want to deprecate the `SimpleTable` since we don't want to maintain two tables.
But my understanding was that we are not doing that in this PR because there are still places where `SimpleTable` is used, so this PR has been about introducing the new table + changing the `ResourceTable` implementation.
I am fine with merging this and moving on to deprecate the `SimpleTable`.